streaming/java/streaming-runtime/src/test/java/org/ray/streaming/runtime/BaseUnitTest.java

```diff
@@ -0,0 +1,34 @@
+package org.ray.streaming.runtime;
+
+import java.lang.reflect.Method;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+public abstract class BaseUnitTest {
+
+  private static final Logger LOG = LoggerFactory.getLogger(BaseUnitTest.class);
+
+  @BeforeClass
+  public void setUp() {
+    TestHelper.setUTPattern();
+  }
+
+  @AfterClass
+  public void tearDown() {
+    TestHelper.clearUTPattern();
+  }
+
+  @BeforeMethod
+  public void testBegin(Method method) {
```

Comment:
Shall we always run testBegin and testEnd?

Reply:
Not essential. But adding this output is really good for problems positioning when debug unit tests' logs on server.